cmd/gardener-resource-manager/app/options.go

```diff
@@ -14,26 +14,21 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 
 	"github.com/gardener/gardener/cmd/utils/initrun"
-	"github.com/gardener/gardener/pkg/resourcemanager/apis/config"
 	resourcemanagerconfigv1alpha1 "github.com/gardener/gardener/pkg/resourcemanager/apis/config/v1alpha1"
-	resourcemanagervalidation "github.com/gardener/gardener/pkg/resourcemanager/apis/config/validation"
+	resourcemanagervalidation "github.com/gardener/gardener/pkg/resourcemanager/apis/config/v1alpha1/validation"
```

Comment:
Shouldn't we also need an import alias for versioned validation, similar to `configv1alpha1`?

Reply:
I also considered this question. I ended up not introducing aliases for the validation packages because there will always be one validation package per API group – not multiple validation packages in different versions. Hence, this package could be considered _the_ validation package of the group, so leaving out the version made sense to me :)